teps/0098-workflows.md

```diff
@@ -256,11 +306,164 @@ Why should this TEP _not_ be implemented?
+[hub resolution](https://pipelinesascode.com/docs/guide/resolver/#tekton-hubhttpshubtektondev) and the built-in git resolver rather than using
+the Pipelines as Code [remote URL syntax](https://pipelinesascode.com/docs/guide/resolver/#remote-http-url).
+(Note: this work is already planned for Pipelines as Code.)
+1. Create a Workflow CRD that allows users to configure much of the behavior that's currently configured through annotations, such as which events
+trigger a PipelineRun and concurrency controls to apply. We may choose to rework the existing implementation so that configuring a PipelineRun
+with annotations would be implemented by creating a Workflow under the hood. We'd support the existing API (PipelineRun with annotations) for at
+least 6 months, and then we can reevaluate how to reduce duplication and create a more consistent API while still maintaining an easy setup experience
+for users who want to configure workflows only via their Git repo rather than on the Kubernetes cluster.
+1. Set a long term goal of updating the project to use Triggers. The project used to be based on Triggers (for more info, see the 
+[latest branch](https://github.com/openshift-pipelines/pipelines-as-code/tree/release-0.5.0/config) using a Triggers backend),
+but migrated to a custom implementation due to several of the [pain points](#use-of-triggers) identified above.
+We'll have to fix these pain points before reworking the implementation to use Triggers, and the refactoring may be challenging to do incrementally
+or not worth the time required to do so. We'll have to weigh the difficulty of refactoring against the difficulty of maintaining
+similar code in both projects.
+1. Explore building extensibility mechanisms into the project. It's not yet clear what this would look like.
+
+### Adopt Pipelines as Code alongside Workflows
+
+In this solution, the Tekton community would adopt and maintain the Pipelines as Code project, but not
+use it as a Workflows replacement. Instead, it could serve as an example of how to build a CI/CD platform
+on top of Tekton Pipelines. Pipelines as Code can continue to focus on an approach where Pipeline definitions
+are stored in the repository that is the subject of a CI/CD workflow, and can be tested by creating a pull request
+on the repository, while Workflows can focus on an approach more similar to other Tekton projects,
+where a cluster operator installs the Workflows project and configures repo connections and an app developer
+defines Workflows by creating CRDs on a cluster.
+
+Because these projects have similar goals, this solution risks creating redundant work in the Tekton community
+and causing confusion to users about which project they should use for CI/CD. In addition, Pipelines as Code
+doesn't use Triggers, so it's not clear whether we'd be encouraging platform builders to use Triggers or not.
```

Comment:
Triggers is way more flexible and less opinionated than PAC.  

The current PAC design is really made to be as "close" as possible to the end user code making it easy to get started, see statuses, rerun etc..... directly from the GIT provider platform...

That don't necessary how every admins/users want to run their CI/CD and that's where triggers may fill that role.